projects/angular/src/forms/styles/_mixins.forms.scss

```
@@ -116,17 +110,12 @@
   margin-top: 0;
   cursor: pointer;
   min-height: $min-height;
-  padding: #{tokens.$cds-global-space-3} 0 #{tokens.$cds-global-space-3} $padding-left;
+  padding: #{tokens.$cds-global-space-3} 0;
   color: $color;
   @include mixins.generate-typography-token($typography);
 }
 
-@mixin checkbox-radio-shared-inactive($size, $top, $left, $border-color) {
-  //Positioning
-  position: absolute;
-  top: $top;
-  left: $left;
-
+@mixin checkbox-radio-shared-inactive($size, $border-color) {
```

Position absolute is needed to neagate parent's line-height.

```suggestion
@mixin checkbox-radio-shared-inactive($size, $border-color) {
  //Positioning
  position: absolute;
```